docker-compose.yml

```diff
@@ -253,6 +254,11 @@ services:
     ports:
       - 19531:3000
     profiles: [ all, infra ]
+
```

Comment:
Please add it to the `infra` profile, we don't have a path or documentation for users to benefit from the collector so I don't want to take up extra memory and download time by this.

Or we could extract observe + mlflow + collector to as separate `start:observe` profile, most people don't need these by default

Reply:
> we could extract observe + mlflow + collector to as separate start:observe profile

+1 like that idea a lot